scripts/README.md

```diff
@@ -0,0 +1,30 @@
+rosprolog_rest
+===
+
+The ros node rosprolog_rest provides an REST API for KnowRob.
+
+### Installation instructions
+
+1. Install the following python packages:
+'''
+sudo -H pip3 install flask_restplus gevent
+'''
+
+2. Run this code to fix some known bugs:
+'''
+sudo sed -i 's/werkzeug import cached_property/werkzeug.utils import cached_property/' /usr/local/lib/python*/dist-packages/flask_restplus/fields.py
```

Comment:
what is going on here? What are these bugs?
This does not look like it will run for everyone.
Is this really needed?

Reply:
It happen because the library werkzeug changed but that changes weren't updated in the libraries of flask_restplus.
It simply replaces in the libraries **fields.py** and **api.py** of **flask_restplus**:
`from werkzeug import cached_property` to `from werkzeug.utils import cached_property`
You can see that in the libraries

Reply:
And yes, it's really needed, otherwise the werkzeug library must be downgraded

Reply:
But this is not a real solution, but rather a hack. Can we find a better solution?
Because it is not a good idea to suggest users patching installed files (i.e. in /usr/local)

Reply:
It's done